docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-edge-syntax.md

```diff
@@ -1,9 +1,9 @@
 # INSERT EDGE 语法
 
 ```ngql
-INSERT EDGE edge_name ( <prop_name_list> ) {VALUES | VALUE}
+INSERT EDGE <edge_name> ( <prop_name_list> ) VALUES | VALUE
 <src_vid> -> <dst_vid>@ranking : ( <prop_value_list> )
```

Comment:
I think `<ranking>` is better

Reply:
But the syntax requires a @, so I leave it as it is.

Reply:
```
<src_vid> -> <dst_vid>[@<ranking>] : ( <prop_value_list> )
```
but it's ok. fix next time